pubsub/service.go

```diff
@@ -62,6 +62,14 @@ func (r *defaultRetryer) Retry(err error) (pause time.Duration, shouldRetry bool
 			return r.bo.Pause(), true
 		}
 		return 0, false
+	case codes.Unknown:
+		// Retry GOAWAY, see https://github.com/googleapis/google-cloud-go/issues/4257.
+		isGoaway := strings.Contains(s.Message(), "closing error reading from server: EOF") &&
```

Comment:
Just checking: does the error message contain exactly "closing error reading from server..."? I looked at the linked issue and that phrase isn't prefixed by closing, so it may not match.